src/content/advanced/spot-instances/_index.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Spot instances
-description: Spot instances (also called low-priority VMs on Azure) are a simple way to save on compute cost, if your use case permits it.
+description: Spot instances (also called low-priority VMs on Azure) are a simple way to save on compute cost, if your use case permits it. Here you find documentation for using them with Giant Swarm Kubernetes clusters, both on AWS and Microsoft Azure.
 weight: 40
 menu:
   main:
     identifier: advanced-spotinstances
     parent: advanced
 aliases:
-  - /basics/spot-instances/
+  - /advanced/spot-instances/
 ---
 
 {{< platform_support_table aws="ga=v11.2.0" azure="roadmap=https://github.com/giantswarm/roadmap/issues/7" >}}
```

Comment:
This renders as:

![image](https://user-images.githubusercontent.com/273727/107251246-6e853780-6a34-11eb-9443-99a24d35987c.png)

I suggest that you merge this PR at the same time when Azure v14.1.0 gets releases. That means, this line has to change to this (if the feature is considered GA, not alpha or beta):

```suggestion
{{< platform_support_table aws="ga=v11.2.0" azure="ga=v14.1.0" >}}
```